README.md

```diff
@@ -308,7 +308,7 @@ speed up your data preparation work.
 - Data lineage and experiment tracking
 - Distributed execution, both on and off cluster
 - Support other dataframe libraries such as HF Datasets, Polars, Spark
-- Move reusable components into a decentralized component registry
```

Comment:
why was this removed? This is still on the roadmap :)

Reply:
Wasn't this the docker hub?